libs/elements/layout/buttons/src/lib/main/elewa-normal-button/elewa-normal-button.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
```

Comment:
The button should accept input and output a click event. Also, the input should have a type set.